content/kotsadm/access/securing-the-console.md

```diff
@@ -0,0 +1,26 @@
+---
+date: 2020-12-08
+linktitle: "Securing the Console"
+title: Securing access to the KOTS Admin Console
+weight: 1
+draft: false
+isBeta: true
+---
+
+Upon initial installation, the KOTS Admin Console is secured with a single shared password for all users. It is possible to further configure the KOTS Admin Console to authenticate users with your organization's user management system.
+
+KOTS leverages the open-source project [Dex](https://dexidp.io/) as an intermediary to control access to the console. Dex implements an array of protocols for querying other user-management systems, known as [connectors](https://dexidp.io/docs/connectors/).
+
+## Configuration
+
+To begin, click the Access link at the top of the Admin Console. Here you can configure access to the KOTS Admin Console, integrating with one of the supported identity providers.
```

Comment:
should we mention that the license will have to support this option?

Reply:
thank you. adding...